src/feedspora/diaspora_client.py

```diff
@@ -1,6 +1,7 @@
 """
 Diaspora client.
 """
+import copy
```

Comment:
I was mostly being defensive towards unknown future changes.  Also, it could get strange during the call to set_common_opts, which ends up passing the exact same account as an argument (in the non-test scenario, anyway).
Do you still think we should remove the copy operation?

Reply:
I think it's better to make the code complex only when necessary. Since the `account` is loaded from the yaml configuration file, it's not expected to change unless there's a major change in how feedspora works. If it's the case in the future, we'll change it. I think it's better to remove it for now.

Reply:
Done